app/src/main/java/com/e/salesapp/MainActivity.java

```diff
@@ -1,44 +1,31 @@
 package com.e.salesapp;
 
-import android.content.Context;
 import android.content.Intent;
-import android.inputmethodservice.Keyboard;
 import android.os.Bundle;
-import android.util.Log;
 import android.view.Menu;
-import android.view.View;
-
 import androidx.annotation.NonNull;
 import androidx.core.view.GravityCompat;
 import androidx.appcompat.app.ActionBarDrawerToggle;
 import android.view.MenuItem;
-import android.widget.FrameLayout;
-
-import com.e.salesapp.adapters.ProductAdapter;
 import com.google.android.material.navigation.NavigationView;
-
 import androidx.drawerlayout.widget.DrawerLayout;
-
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.appcompat.widget.Toolbar;
 import androidx.fragment.app.FragmentManager;
-import androidx.recyclerview.widget.LinearLayoutManager;
-import androidx.recyclerview.widget.RecyclerView;
-
-import java.util.ArrayList;
-import java.util.List;
 
 public class MainActivity extends AppCompatActivity
-        implements  NavigationView.OnNavigationItemSelectedListener, CategoriesListFragment.OnCategorySelected{
+        implements  NavigationView.OnNavigationItemSelectedListener,
+                    CategoriesListFragment.OnCategorySelected {
+
+    private FragmentManager mFragmentManager;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main_menu);
-
-        FragmentManager manager = getSupportFragmentManager();
-        manager.beginTransaction()
-                .replace(R.id.container, new CategoriesListFragment(this))
+        mFragmentManager = getSupportFragmentManager();
```

Comment:
is necessary puth this methods inside "onCreate" method?, why you dont created a private method and puth inside it this methods?, it would make more readable your code(it just a suggestion)